source/Ship.cpp

```diff
@@ -859,7 +861,7 @@ void Ship::Save(DataWriter &out) const
 			using OutfitElement = pair<const Outfit *const, int>;
 			WriteSorted(outfits,
 				[](const OutfitElement *lhs, const OutfitElement *rhs)
-					{ return lhs->first->Name() < rhs->first->Name(); },
+					{ return lhs->first->TrueName() < rhs->first->TrueName(); },
 				[&out](const OutfitElement &it){
 					if(it.second == 1)
 						out.Write(it.first->Name());
```

Comment:
```suggestion
						out.Write(it.first->TrueName());
```

and the one just below also.